### DISCUSSION TITLE
Replace "Show in-progress rooms" in multiplayer with "Show locked rooms"

### DISCUSSION THREAD

**Discussion by behsn** [ppy/osu#31204] [Ideas] [answered]
Or having it as a separate option would be nice, but being honest I really don't see the point of hiding in-progress rooms, with how fast lobbies begin, how short songs are, and how few lobbies there are at a given time, it just seems like a useless feature. 

Hiding locked rooms on the other hand would be nice for people playing public lobbies who have to scroll through endless locked 2 person lobbies to find a single public one.

**Comment by Joehuu** [comment, marked as answer]
Changing this to public/private is equivalent to hiding / only showing locked rooms respectively.

![osu!_EsjOYiX3aR](https://github.com/user-attachments/assets/ae656b80-2af9-474e-bde9-feb375d1993d)

**Comment by bdach** [comment]
> it just seems like a useless feature

https://osu.ppy.sh/community/forums/topics/2013293 begs to differ.
